### PR TITLE
Issue377 hvac agent based control

### PR DESCRIPTION
**Pull request by FelixBue**
Regarding issue #377 

This integrates the HVACAgentBasedControl library into the controls package of the AixLib.

### CODE REVIEW

File: AixLib/Controls/HVACAgentBasedControl/Examples/BuildingHeatingSystems/BuildingWithPV.mo [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+<p>This model was used to present the HVACAgentBasedControl library in [Roozbeh Sangi, Felix B&uuml;nning, Marc Baranski, Johannes F&uuml;tterer, Dirk M&uuml;ller. A Platform for the Agent-based Control of HVAC Systems. Modelica Conference, 2017, Prague, Czech Republic]. For detailed information please refer to this source. </p>
+<h4><span style=\"color: #008000\">Example Results</span></h4>
+<p>The results generated by this model differ from the results presented in [Roozbeh Sangi, Felix B&uuml;nning, Marc Baranski, Johannes F&uuml;tterer, Dirk M&uuml;ller. A Platform for the Agent-based Control of HVAC Systems. Modelica Conference, 2017, Prague, Czech Republic] because an older version of the AixLib library was used. The model presented here was adapted to the use of the most recent AixLib library, which has a different physical model for the thermal zones than the version used originally. The result image below (generated with this model) however gives an impression on how the system behaves depneding on the price of electricity.</p>
```
Comment (marcusfuchs) [inline AixLib/Controls/HVACAgentBasedControl/Examples/BuildingHeatingSystems/BuildingWithPV.mo]: Now that this package is part of AixLib, `was adapted to the use of the most recent AixLib library` may sound a bit confusing to users.

**Comment by marcusfuchs** [comment]
@FelixBue @RoozbehSangi 
This is really great work! Could you fix what you think makes sense in my remarks and assign back to me when you think this is ready to merge?

File: AixLib/Controls/HVACAgentBasedControl/UsersGuide/Overview.mo [hunk 2]
```diff
@@ -0,0 +1,17 @@
+within AixLib.Controls.HVACAgentBasedControl.UsersGuide;
+class Overview "Overview"
+  extends Modelica.Icons.Information;
+  annotation (Documentation(info="<html>
+<p><span style=\"font-family: Arial,sans-serif;\">The HAVC Agent-based Control Library consists of the following main sub-libraries: </span></p>
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/UsersGuide/Overview.mo]: Done!

File: AixLib/Controls/HVACAgentBasedControl/UsersGuide/Requirements.mo [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+class Requirements "Requirements"
+  extends Modelica.Icons.Information;
+  annotation (Documentation(info="<html>
+<p>
+In order to use the components provided in this library, the following additional (free) libraries are needed:
+</p>
+<p>
+<ul>
+<li>Modelica_DeviceDrivers library in release version \"1.4.4\" (https://github.com/modelica/Modelica_DeviceDrivers)</li>
+<li>AixLib library in release version \"0.3.2\" (https://github.com/RWTH-EBC/AixLib) in order to run the example models</li>
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/UsersGuide/Requirements.mo]: Done!

File: AixLib/Controls/HVACAgentBasedControl/UsersGuide/Gettingstarted.mo [hunk 4]
```diff
@@ -0,0 +1,43 @@
+within AixLib.Controls.HVACAgentBasedControl.UsersGuide;
+class Gettingstarted "Getting started"
+  extends Modelica.Icons.Information;
+  annotation (Documentation(info="<html>
+<h4><span style=\"color: #008000\">What you can do with this library</span></h4>
+<p>The Modelica HVACAgendBasedControl presents a set of agents which can control building energy systems and systems of similar characteristics (e.g. district heating
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/UsersGuide/Gettingstarted.mo]: Done!

File: AixLib/Controls/HVACAgentBasedControl/UsersGuide/Gettingstarted.mo [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+</p>
+<p>
+The system is now ready to go.
+</p>
+<p><b>
+If you used older versions of this library, please add a MessageNotification agent to your agent-based systems in order to make them work. For more information please refer to
+<a href=\"HVACAgentBasedControl.Agents.MessageNotification\">MessageNotification model</a>.
+</b></p>
+<h4><span style=\"color: #008000\">What can a building energy system with agent-based control look like?</span></h4>
+<p>Below you see an example building energy system which can be found <a href=\"HVACAgentBasedControl.Examples.SimpleHeatingSystem.BuildingHeating\">here</a>.
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/UsersGuide/Gettingstarted.mo]: Done!

File: AixLib/Controls/HVACAgentBasedControl/UsersGuide/ModelicaLicense2.mo [hunk 6]
```diff
@@ -0,0 +1,14 @@
+within AixLib.Controls.HVACAgentBasedControl.UsersGuide;
+class ModelicaLicense2 "Modelica License 2"
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/UsersGuide/ModelicaLicense2.mo]: Done!

File: AixLib/Controls/HVACAgentBasedControl/UsersGuide/Copyright.mo [hunk 7]
```diff
@@ -0,0 +1,12 @@
+within AixLib.Controls.HVACAgentBasedControl.UsersGuide;
+class Copyright "Copyright"
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/UsersGuide/Copyright.mo]: Done!

File: AixLib/Controls/HVACAgentBasedControl/UsersGuide/Releasenotes/Version_0_1_0.mo [hunk 8]
```diff
@@ -0,0 +1,14 @@
+within AixLib.Controls.HVACAgentBasedControl.UsersGuide.Releasenotes;
+class Version_0_1_0 "Version 0.1.0 (November 15, 2016)"
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/UsersGuide/Releasenotes/Version_0_1_0.mo]: removed!

File: AixLib/Controls/HVACAgentBasedControl/UsersGuide/Contact.mo [hunk 9]
```diff
@@ -0,0 +1,21 @@
+within AixLib.Controls.HVACAgentBasedControl.UsersGuide;
+model Contact "Contact"
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/UsersGuide/Contact.mo]: Package removed!

File: AixLib/Controls/HVACAgentBasedControl/Agents/MessageNotification.mo [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+<p>Every agent system needs a MessageNotification model in order to function, similar to the \"System\" model of the Modelica.Fluid library. The MessageNotification is responsible for ensuring that the agents receive their messages
+and update their inbox.</p>
+<p>There are two different possibilities to notify the agents when they receive a message. The standard one corresponds to earlier versions of the library and is set by setting usePoke=false in the MessageNotification models
+and all other agents in the system. When this setting is made, the UDP inbox of all agents gets updated every n seconds (n set by user). This is straight forward but also leads to time event generation (weak simulation 
+performance). If usePoke is set to true, the Boolean sendOut connectors of every agent in the system needs to be connected to the Boolean receive[n] input of the MessageNotification model. With this method, the agents only update
+their inbox in case one agent has send out a message. This reduces event generation during idle times of the agents and thus may increase simulation perfomance in systems with long idle times of the agents. Example images
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/Agents/MessageNotification.mo]: Modified!

File: AixLib/Controls/HVACAgentBasedControl/Agents/HeatProducerAgent.mo [hunk 11]
```diff
@@ -0,0 +1,660 @@
...
+<p><img src=\"modelica://AixLib/Resources/Images/Controls/HVACAgentBasedControl/ProducerAgent.png\"/></p>
+<h4><span style=\"color: #008000\">References</span></h4>
+<ul>
+<li>Roozbeh Sangi, Felix B&uuml;nning, Marc Baranski, Johannes F&uuml;tterer, Dirk M&uuml;ller. A Platform for the Agent-based Control of HVAC Systems. Modelica Conference, 2017, Prague, Czech Republic.</li>
+<li>FIPA ACL Message Structure Specification</li>
+<li>FIPA Communicative Act Library Specification </li>
+</ul>
+<h4><span style=\"color: #008000\">Example Results</span></h4>
+<ul>
+<li><a href=\"HVACAgentBasedControl.Examples.SimpleHeatingSystem.BuildingHeating\">Simple Heating System</a></li>
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/Agents/HeatProducerAgent.mo]: Updated!

File: AixLib/Controls/HVACAgentBasedControl/Agents/MessageNotification.mo [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+their inbox in case one agent has send out a message. This reduces event generation during idle times of the agents and thus may increase simulation perfomance in systems with long idle times of the agents. Example images 
+for both methods are provided below. For further information you can also refer to the example results.</p>
+<p>System with usePoke=false</p>
+<p><img src=\"modelica://AixLib/Resources/Images/Controls/HVACAgentBasedControl/ExampleSystemMarked.png\"/></p>
+<p>System with usePoke=true</p>
+<p><img src=\"modelica://AixLib/Resources/Images/Controls/HVACAgentBasedControl/ExampleSystemMarkedPoke.PNG\"/></p>
+<h4><span style=\"color: #008000\">Example Results</span></h4>
+<ul>
+<li><a href=\"HVACAgentBasedControl.Examples.SimpleHeatingSystem.BuildingHeating\">System with usePoke=false</a></li>
+<li><a href=\"HVACAgentBasedControl.Examples.SimpleHeatingSystem.BuildingHeating_usePoke\">System with usePoke=true</a></li>
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/Agents/MessageNotification.mo]: Updated!

File: AixLib/Controls/HVACAgentBasedControl/Agents/ColdProducerAgent.mo [hunk 13]
```diff
@@ -0,0 +1,735 @@
...
+<p><img src=\"modelica://AixLib/Resources/Images/Controls/HVACAgentBasedControl/ProducerAgent.png\"/></p>
+<h4><span style=\"color: #008000\">References</span></h4>
+<ul>
+<li>Roozbeh Sangi, Felix B&uuml;nning, Marc Baranski, Johannes F&uuml;tterer, Dirk M&uuml;ller. A Platform for the Agent-based Control of HVAC Systems. Modelica Conference, 2017, Prague, Czech Republic. </li>
+<li>FIPA ACL Message Structure Specification</li>
+<li>FIPA Communicative Act Library Specification </li>
+</ul>
+<p><b><span style=\"font-family: MS Shell Dlg 2; color: #008000;\">Example Results</span></b></p>
+<ul>
+<li><a href=\"HVACAgentBasedControl.Examples.SimpleHeatingSystem.BuildingHeating\">Simple Heating System</a></li>
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/Agents/ColdProducerAgent.mo]: Updated!

File: AixLib/Controls/HVACAgentBasedControl/Agents/ColdProducerAgent.mo [hunk 14]
```diff
@@ -0,0 +1,735 @@
...
+<ul>
+<li>This model is a producer agent which controls a cold producing device./li&GT; </li>
+<li>It is based on communication via UDP and logic implemented with the help of the StateGraph Modelica library.</li>
+<li>It is used together with a broker-agent and at least one producer-agent.</li>
+<li>It requires a CostFunction component to compute a proposal.</li>
+</ul>
+<h4><span style=\"color: #008000\">Concept</span></h4>
+<p>The heat producer agent recives a call for proposal from the broker with a requested capacity adjustment. It makes a proposal with the help of a Cost function for the adjustment and sends it back to the broker. If the proposal is accepted, the agent confirms the adjustment and adjusts the capacity of the controlled device accordingly. The logic is implemented with the help of the StateGraph library. Communication is realized with the help of the DeviceDriver library and follows the language standards for multi-agent-systems set by the FIPA to the highest possible extend for Modelica models. The following figure shows the behaviour of the consumer agent. For further information please refer to the first reference.</p>
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/Agents/ColdProducerAgent.mo]: Modified!

File: AixLib/Controls/HVACAgentBasedControl/Agents/ConsumerAgent.mo [hunk 15]
```diff
@@ -0,0 +1,489 @@
+within AixLib.Controls.HVACAgentBasedControl.Agents;
+model ConsumerAgent
+  extends BaseClasses.PartialAgent(
+    name=10001,
+    uDPSend_adapted(userBufferSize=100000, autoBufferSize=true),
+    uDPReceive_adapted(userBufferSize=100000, autoBufferSize=true));
+
+  parameter Integer broker = 10003 "Name of the corresponding broker-agent";
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/Agents/ConsumerAgent.mo]: It should be as it is.

File: AixLib/Controls/HVACAgentBasedControl/Agents/IntermediateAgent.mo [hunk 16]
```diff
@@ -0,0 +1,689 @@
...
+        Polygon(
+          points={{0,-10},{-22,-10},{-22,-46},{-40,-46},{0,-84},{0,-10}},
+          smooth=Smooth.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None)}),
+    Documentation(info="<html>
+<h4><span style=\"color: #008000\">Overview</span></h4>
+<ul>
+<li>This model is an intermediate agent which represents devices to connect thermal circuits (heat exchnager, mixing valve).</li>
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/Agents/IntermediateAgent.mo]: Modified!

File: AixLib/Controls/HVACAgentBasedControl/Agents/IntermediateAgent.mo [hunk 17]
```diff
@@ -0,0 +1,689 @@
...
+<li>It is used together with a broker-agent and at least one producer-agent.</li>
+</ul>
+<h4><span style=\"color: #008000\">Concept</span></h4>
+<p>The intermediate agent acts as a producer in front of one broker (bottom) and consumer in front of another broker (top). When it gets a call for proposal from the bottom broker, it passes it on to the top broker as an request. This broker handels the request and responds with a price information. The intermediate agent adds its own costs and passes it on as a proposal to the bottom broker. The proposal gets either rejected or accepted. This is passed on a a confirm/reject to the top broker. The intermediate agent also adjusts it capacity according to the requested adjustment. The logic is implemented with the help of the StateGraph library. Communication is realized with the help of the DeviceDriver library and follows the language standards for multi-agent-systems set by the FIPA to the highest possible extend for Modelica models. The following figure shows the behaviour of the roomagent. For further information please refer to the first reference.</p>
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/Agents/IntermediateAgent.mo]: Modified!

File: AixLib/Controls/HVACAgentBasedControl/Agents/RoomAgent.mo [hunk 18]
```diff
@@ -0,0 +1,537 @@
...
+      pattern=LinePattern.Dot));
+  annotation (Documentation(info="<html>
+<h4><span style=\"color: #008000\">Overview</span></h4>
+<ul>
+<li>This model is a room-agent which controls the temperature inside the room.</li>
+<li>It is based on communication via UDP and logic implemented with the help of the StateGraph Modelica library.</li>
+<li>It is used together with a broker-agent and at least one producer-agent.</li>
+</ul>
+<p><b><span style=\"font-family: MS Shell Dlg 2; color: #008000;\">Concept</span></b></p>
+<p>The room agent observes the temperature inside a room. If the temperature crosses a certain treshhold and a pre-defined sample-time has elapsed, a cooling or heating request is calculated and send to a broker agent. The broke-agent ensures that the requested heat is produced. The logic is implemented with the help of the StateGraph library. Communication is realized with the help of the DeviceDriver library and follows the language standards for multi-agent-systems set by the FIPA to the highest possible extend for Modelica models. The following figure shows the behaviour of the roomagent. For further information please refer to the first reference.</p>
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/Agents/RoomAgent.mo]: Modified!

File: AixLib/Controls/HVACAgentBasedControl/Agents/RoomAgent.mo [hunk 19]
```diff
@@ -0,0 +1,537 @@
...
+<p><img src=\"modelica://AixLib/Resources/Images/Controls/HVACAgentBasedControl/RoomAgent.png\"/></p>
+<h4><span style=\"color: #008000\">References</span></h4>
+<ul>
+<li>Roozbeh Sangi, Felix B&uuml;nning, Marc Baranski, Johannes F&uuml;tterer, Dirk M&uuml;ller. A Platform for the Agent-based Control of HVAC Systems. Modelica Conference, 2017, Prague, Czech Republic.  </li>
+<li>FIPA ACL Message Structure Specification</li>
+<li>FIPA Communicative Act Library Specification </li>
+</ul>
+<h4><span style=\"color: #008000\">Example Results</span></h4>
+<ul>
+<li><a href=\"HVACAgentBasedControl.Examples.SimpleHeatingSystem.BuildingHeating\">Simple Heating System</a></li>
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/Agents/RoomAgent.mo]: Updated!

File: AixLib/Controls/HVACAgentBasedControl/Agents/Broker.mo [hunk 20]
```diff
@@ -0,0 +1,1091 @@
...
+      points={{-65.5,-336},{-182,-336},{-182,-446},{-182,-444.857},{-167,
+          -444.857}},
+      color={255,0,0},
+      pattern=LinePattern.Dot));
+
+  annotation (Documentation(info="<html>
+<h4><span style=\"color: #008000\">Overview</span></h4>
+<ul>
+<li>This model is a broker agent which collects requests from room-agent and buys heat for the lowest price from producer agents.</li>
+<li>It is based on communication via UDP and logic implemented with the help of the StateGraph Modelica library./li&GT; </li>
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/Agents/Broker.mo]: Removed!

File: AixLib/Controls/HVACAgentBasedControl/Agents/Broker.mo [hunk 21]
```diff
@@ -0,0 +1,1091 @@
...
+<p><img src=\"modelica://AixLib/Resources/Images/Controls/HVACAgentBasedControl/Broker.png\"/></p>
+<h4><span style=\"color: #008000\">References</span></h4>
+<ul>
+<li>Roozbeh Sangi, Felix B&uuml;nning, Marc Baranski, Johannes F&uuml;tterer, Dirk M&uuml;ller. A Platform for the Agent-based Control of HVAC Systems. Modelica Conference, 2017, Prague, Czech Republic. </li>
+<li>FIPA ACL Message Structure Specification</li>
+<li>FIPA Communicative Act Library Specification </li>
+</ul>
+<h4><span style=\"color: #008000\">Example Results</span></h4>
+<ul>
+<li><a href=\"HVACAgentBasedControl.Examples.SimpleHeatingSystem.BuildingHeating\">Simple Heating System</a></li>
```
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/Agents/Broker.mo]: Updated!

File: AixLib/Controls/HVACAgentBasedControl/Examples/BuildingHeatingSystems/BuildingWithPV.mo (hunk 1 above)
Comment (RoozbehSangi) [inline AixLib/Controls/HVACAgentBasedControl/Examples/BuildingHeatingSystems/BuildingWithPV.mo]: Modified!

**Comment by RoozbehSangi** [comment]
@marcusfuchs Thank you for your comments! I've applied all your comments except the general remarks and questions at the beginning.  The rest can be done after merging.

**Comment by marcusfuchs** [comment]
@RoozbehSangi Thanks for fixing the inline comments. We can postpone most of my general comments, but in my opinion we should at least clarify points 6. and 8. before merging. What is your take on these issues?

**Comment by FelixBue** [comment]
@marcusfuchs and @RoozbehSangi As I don't have a Dymola license anymore, I can't really work on the library anymore. But here are my thoughts regarding the two points anyway:

6: I Don't see any problem with changing the mixing volume.
8: When I adapted the example to the current AixLib version, the model of the thermalZone changed and it seemed like there was only a record for one type of room available ("OPH_1_Office()" I think). The results in the documentation, however, were done with one record of "OPH_1_Meeting()" and one of "OPH_4_Restroom()". This leads to the different results.

**Comment by marcusfuchs** [comment]
@FelixBue Thanks for the feedback. It seems like the other example records were removed because they were not needed. But I suggest to bring them back, as this example shows that they can help to demonstrate control concepts with more interesting results. I'll try to re-integrate that.

**Comment by RoozbehSangi** [comment]
@marcusfuchs Totally agree, but since we were supposed to realse a new version by the end of May, I did as much as I could before going on vacation.

**Comment by RoozbehSangi** [comment]
@marcusfuchs Below you find answers to your 8 questions:

1.	Not all of the agents, but the ones that need to have that value already have it.
2.	As far as I see, all the parameters used in the cost functions already have default values.
3.	The documentation revised.
4.	This comment can be applied after merging.
5.	All comments within the code modified accordingly.
6.	The warning eliminated.
7.	The stop time reduced.
8.	The problem was with the internal heat gain model in the example. The figures in the documentation were generated by the initial example, which works with AixLib 0.3.2, but the example experienced a few unwanted changes on the integration way to AixLib 0.4.0. The example has now been modified, but the simulation is still running. I’ll update the documentation with the new figures on Monday before merging.

I think we can integrate the Agent library after adding the new figures.

**Comment by marcusfuchs** [comment]
@RoozbehSangi Thanks for the changes!

Regarding 1. and 2.: I see that my questions were not clearly formulated, sorry for that! I was wondering whether it would be better to remove the default values, as that would force users to notice these parameters and set them according to their application. But as I understand it, we will keep the default values for this merge.

Regarding 4.: Could you then create one or more issues for this and tag them with the `good first issue` tag? (see #427 and e.g. #373) 

Regarding 8.: This means that #412 was not the reason and should be put on hold or closed? Looking forward to the revised documentation!

In addition, your latest commit added many white-space-changes in unrelated models. While these seem to improve the code, they should not be included in this pull request. I will try to undo them. 

While at it, I will also address the warnings about invalid utf-8 encodings in Dymola 2018.

**Comment by marcusfuchs** [comment]
@RoozbehSangi Looking at your last commit, I would think that only the file `AixLib/Controls/HVACAgentBasedControl/Examples/BuildingHeatingSystems/BuildingHeating.mo` was modified and that I can simply reset the remaining 119 files? I will do that in a separate branch and create a pull request to your branch so you can double-check again.

**Comment by marcusfuchs** [comment]
I fixed the above mentioned points in the branch [issue377_patch](https://github.com/RWTH-EBC/AixLib/tree/issue377_patch). Still we have lot of problems with the html as shown by Travis after merging the development: https://travis-ci.org/RWTH-EBC/AixLib/builds/256812333?utm_source=github_status&utm_medium=notification
I'll start fixing this...

**Comment by marcusfuchs** [comment]
@RoozbehSangi It seems like all your author informations in the revision notes cause html errors. Please in the future use the sample code as described in [the author information part of the Wiki](https://github.com/RWTH-EBC/AixLib/wiki/Documentation#author-information)

**Comment by marcusfuchs** [comment]
@RoozbehSangi 
I was able to narrow down the html errors to this state: https://travis-ci.org/RWTH-EBC/AixLib/builds/256829540?utm_source=github_status&utm_medium=notification
Unfortunately, I have to stop here. Could you please continue with the following:

- Check wether my branch `issue377_patch` contains all relevant changes from your last commit. If not, please transfer the changes I missed and then continue with this branch.
- Add alternative text tags to all images in the documentation and fix the remaining html errors.
- Add the changes for remark 8. (new images of the example results)
- Coordinate with @AnaConstantin regarding merging the `ìssue377_patch` branch into the release candidate branch
- After the release candidate branch is in master and development, I think we can close this PR and the corresponding issue (pending the creation of a new issue for remark 4.)

**Comment by marcusfuchs** [comment]
@RoozbehSangi I assume you have changed plans and we will not update the images, but merge without images in the documentation of the examples? I will prepare a new PR from the patch branch to the release candidate branch. In case images will be added nonetheless, we can do so in a separate issue after the release. For this PR, I suggest closing after we have merged the patch into the release and @RoozbehSangi has created the issue(s) for remark 4.

**Comment by RoozbehSangi** [comment]
@marcusfuchs Your assumption is right. I discussed this issue with @AnaConstantin , and we concluded that results figures should not be included in the documentation, as they may change in each new release. I’m making a new issue for remark 4. Thank you for handling the HTML errors!

**Comment by mlauster** [comment]
This has been transferred to antoher PR and will be closed

**Pull request closed without merging**